server/sonar-server/src/main/java/org/sonar/server/metric/ws/UpdateAction.java

```diff
@@ -97,11 +95,10 @@ public void define(WebService.NewController context) {
 
   @Override
   public void handle(Request request, Response response) throws Exception {
-    userSession.checkLoggedIn().checkPermission(GlobalPermissions.SYSTEM_ADMIN);
+    userSession.checkLoggedIn().checkIsRoot();
```

Comment:
description should be fixed, root is now required, not system administration... or is that the name we give to root?